showcase/blueprints/hds-component-test/files/tests/integration/components/hds/__name__/index-test.js

```diff
@@ -4,11 +4,13 @@
  */
 
 import { module, test } from 'qunit';
-import { setupRenderingTest } from 'ember-qunit';
+import { setupRenderingTest } from 'showcase/tests/helpers';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 
-module('Integration | Component | hds/<%= dasherizedModuleName %>/index', function (hooks) {
+module(
```

Comment:
[rant] this new linting/formatting is... 🤬 [/rant]

Not your fault, it happens also in my VSCode too 🤷‍♂️